code/unclassified/src/average/Average.java

```diff
@@ -0,0 +1,37 @@
+import java.util.*;
+// Part of Cosmos by OpenGenus Foundation
+class Average
+{
+    /**
+    * @param arr array of integers to get the average of.
+    * @param n number of elements.
+    * @return the average calculated as sum(arr)/n
+    */
+    static double avrg(Vector<Integer> arr)
```

Comment:
I feel like shortening the name would me confusing. Additionally, the more common container used for algorithms is `ArrayLists`, so please do that!